app/models/test.rb

```diff
@@ -1,3 +1,5 @@
 class Test < ApplicationRecord
-  belongs_to :category
-end
+  def self.sort_reverse(category)
+    joins("JOIN categories ON tests.category_id = categories.id").where("categories.title = :category", category: category).order(title: :desc).pluck(:title)
```

Comment:
```suggestion
    joins("JOIN categories ON tests.category_id = categories.id").where(categories: { title: category }).order(title: :desc).pluck(:title)
```
рекомендуется использовать хеш-синтаксис, где это уместно
он лаконичней и проще править